dao/src/main/java/greencity/entity/Habit.java

```diff
@@ -48,6 +48,12 @@ public class Habit {
     @Column(name = "default_duration", nullable = false)
     private Integer defaultDuration;
 
+    @Column(name = "is_custom_habit", nullable = false)
+    private Boolean isCustomHabit;
```

Comment:
Do we need it?
I do not see where we use it

Reply:
This field is set for the future so that we can later check if the custom habit.